polkadot/node/network/collator-protocol/src/validator_side/tests/mod.rs

```diff
@@ -506,138 +507,6 @@ async fn advertise_collation(
 	.await;
 }
 
-// As we receive a relevant advertisement act on it and issue a collation request.
-#[test]
-fn act_on_advertisement() {
```

Comment:
Initially the tests in mod.rs were without async backing while the ones in prospective_parachains.rs - with.
After removing ProspectiveParachainsMode all the tests are with async backing enabled and we have got overlaping cases.

In this particular case the test verifies a happy path that an advertisement leads to a fetch. This scenario is covered in other tests (e.g. `second_multiple_candidates_per_relay_parent`, `accept_advertisements_from_implicit_view`, etc). 

Since it doesn't bring any value after the test merge I decided to remove it.